features/events/syslog/src/main/java/org/opennms/netmgt/syslogd/SyslogConnection.java

```diff
@@ -160,6 +162,12 @@ public SyslogProcessor call() {
         try {
             LOG.debug("Converting syslog message into event ({} bytes)", m_bytes.remaining());
 
+            try {
+                SyslogdIPMgrJDBCImpl.getInstance().dataSourceSync();
```

Comment:
You do not need to address this review comment. I am rewriting the IP address manager classes and consolidating them all in one project so this change is no longer necessary.